### ISSUE TITLE
PYTHON_LIBRARY is not properly passed to SIRF, due to SuperBuild CMake bug

### ISSUE THREAD

**Issue by AnderBiguri** [SyneRBI/SIRF-SuperBuild#472]
Read https://github.com/SyneRBI/SIRF-SuperBuild/issues/467#issuecomment-760933231 and subsequent comments for clarification and explanation.

**Comment by KrisThielemans** [comment]
[CMake doc](https://cmake.org/cmake/help/latest/module/FindPythonLibs.html) says
```
If you’d like to specify the installation of Python to use, you should modify the following cache variables:

PYTHON_LIBRARY             - path to the python library
PYTHON_INCLUDE_DIR         - path to where Python.h is found
```
However, that obviously implies that there's only 1 library (not both a release and debug library).

See also #313

**Comment by KrisThielemans** [comment]
In the spirit of backwards compatibility, and until we fix #313, I suggest we pass `PYTHON_LIBRARY` as we currently do only if `PYTHON_LIBRARIES` is actually an existing file. That way I don't think it can do any harm
